polaris/dataset/_base.py

```diff
@@ -80,6 +83,7 @@ class BaseDataset(BaseArtifactModel, abc.ABC):
     # Private attributes
     _zarr_root: zarr.Group | None = PrivateAttr(None)
     _zarr_data: MutableMapping[str, np.ndarray] | None = PrivateAttr(None)
+    _zarr_md5sum_manifest: list[ZarrFileChecksum] = PrivateAttr(default_factory=list)
```

Comment:
**Comment:** The `_zarr_md5sum_manifest` in V1 is - confusingly - different from the `_zarr_manifest_md5sum` in V2. The former is the list of checksums for all files in the Zarr archive, whereas the latter is the checksum of the manifest file. This thus only exists in the V1 class and shouldn't be in the base class.

Reply:
Nice catch.